parser_test.go

```diff
@@ -2673,6 +2673,21 @@ func (s *testParserSuite) TestSessionManage(c *C) {
 	s.RunTest(c, table)
 }
 
+func (s *testParserSuite) TestParseShowOpenTables(c *C) {
+	parser := parser.New()
+
+	tests := []string{
+		`SHOW OPEN TABLES`,
+		`SHOW OPEN TABLES IN test`,
+		`SHOW OPEN TABLES FROM test`,
+	}
+
+	for _, test := range tests {
+		_, _, err := parser.Parse(test, "", "")
```

Comment:
Why not write an `s.RunTest` like the `TestSessionManage` above?

Reply:
refine in https://github.com/pingcap/parser/pull/291/commits/9350018023bbb7d841df9624ce3797a7a1450ced